src/js/moc_js.ml

```diff
@@ -33,8 +33,8 @@ let () =
       method stableCompatible pre post = js_stable_compatible pre post
       method compileWasm mode s = Flags.compiled := true; js_compile_wasm mode s
       method parseCandid s = js_parse_candid s
-      method parseMotoko s = js_parse_motoko s
-      method parseMotokoWithDeps path s = js_parse_motoko_with_deps path s
-      method parseMotokoTyped paths scopeCache = js_parse_motoko_typed paths scopeCache
+      method parseMotoko recovery_enabled s = js_parse_motoko recovery_enabled s
+      method parseMotokoWithDeps recovery_enabled path s = js_parse_motoko_with_deps recovery_enabled path s
+      method parseMotokoTyped recovery_enabled paths scopeCache = js_parse_motoko_typed recovery_enabled paths scopeCache
```

Comment:
Perhaps use camelCase here? Since this is external interface to JS.

Reply:
done